lib/identity/config.rb

```diff
@@ -14,6 +14,10 @@ def dashboard_url
       ENV["DASHBOARD_URL"] || raise("missing=DASHBOARD_URL")
     end
 
+    def sso_base_url
+      ENV["SSO_BASE_URL"] || raise("missing=SSO_BASE_URL")
```

Comment:
I think it would be good to not require this for people who want to run locally or in other environments without this need.